pom.xml

```diff
@@ -329,6 +332,8 @@
                     <excludes>
                         <!-- TEMPORARY since NCE tests sometimes fail to load classes -->
                         <exclude>jmri.jmrix.nce.**</exclude>
+                        <!-- TEMPORARY since locormi tests cause later security policy issues -->
+                        <exclude>jmri.jmrix.loconet.locormi.**</exclude>
```

Comment:
I've found the cause of the security manager failures. ce17627 fixes the issue by not arbitrarily changing the security manager (if in use).